ivy/functional/frontends/tensorflow/signal.py

```diff
@@ -0,0 +1,7 @@
+import tensorflow as tf
```

Comment:
You are supposed to implement this frontend function relying only on the ivy functions, not other frameworks. Please refer to the deep dive document (https://lets-unify.ai/ivy/deep_dive/16_ivy_frontends.html#ivy-frontends) for more details.

Reply:
you can keep the line of "return 0.5 * (1 - alpha) - 0.5 * alpha * ivy.cos(2 * ivy.pi * window_length) " if it produce the correct result(you need to produce a hypothesis test to make sure), while keeping the input arguments as window_length,
 periodic, and 
 dtype.

Reply:
the deep dive document also has a part where it teaches you how to write a test for the frontend function.

Reply:
I had added the hypothesis test function in the test_signal file
below is the code. 


import ivy
from hypothesis import given, strategies as st

# local
import ivy_tests.test_ivy.helpers as helpers
from ivy_tests.test_ivy.helpers import handle_cmd_line_args


# hann_window
@handle_cmd_line_args
@given(
    dtype_and_x=helpers.dtype_and_values(
        available_dtypes=helpers.get_dtypes("float"),
    ),
    num_positional_args=helpers.num_positional_args(
        fn_name="ivy.functional.frontends.tensorflow.hann_window",
    ),
)
# hann_window
def test_hann_window(dtype_and_x, num_positional_args):
    dtype, x = dtype_and_x
    x = helpers.convert_to_tensor(x, dtype=dtype)
    if num_positional_args == 1:
        res = ivy.hann_window(x)
    elif num_positional_args == 2:
        res = ivy.hann_window(x, dtype=dtype)
    else:
        raise ValueError
    assert res.shape == x.shape
    assert res.dtype == x.dtype
    assert res.device == x.device
    assert ivy.allclose(res, ivy.numpy.hann_window(x.numpy(), dtype=dtype.numpy()))

Reply:
It run without getting any errors but i think i am missing out on something.